Eatery Blue/UI/Onboarding/OnboardingFeaturesViewController.swift

```diff
@@ -158,10 +155,18 @@ class OnboardingFeaturesViewController: UIViewController {
         let clampedIndex = max(0, min(pages.count - 1, currentIndex))
         let nextIndex = clampedIndex + 1
 
+        // Mark : Changed nextButton text to "Lets go!" to transition onboarding to HomeModel view
+        
+        if nextIndex == pages.count - 1 {
+            nextButton.content.text = "Lets go!"
+        }
+
         if nextIndex == pages.count {
             // We've reached the last page
-
-            let viewController = OnboardingLoginModelController()
+            
+            // Mark : Deleted login page directly after onboarding
+            
+            let viewController = HomeModelController()
```

Comment:
Can you name this variable something more specific? You can also omit this line and just do `navigationController?.pushViewController(HomeModelController(), animated: true)`.